e2e/active_test.go

```diff
@@ -23,18 +23,22 @@ import (
 
 	"github.com/ossf/scorecard/v2/checker"
 	"github.com/ossf/scorecard/v2/checks"
+	"github.com/ossf/scorecard/v2/clients/githubrepo"
 	scut "github.com/ossf/scorecard/v2/utests"
 )
 
 var _ = Describe("E2E TEST:Active", func() {
```

Comment:
use the `checker.CheckActive` variable `"E2E TEST:" + checker.CheckActive`. I've started doing this too.

Reply:
Done.

Reply:
Done